apis/filesystem/src/test/java/org/jclouds/filesystem/integration/FilesystemBlobIntegrationTest.java

```diff
@@ -41,38 +52,65 @@ protected Properties setupProperties() {
       return props;
    }
 
-   // Mac OS X HFS+ does not support UserDefinedFileAttributeView:
-   // https://bugs.openjdk.java.net/browse/JDK-8030048
+   @Test(dataProvider = "ignoreOnMacOSX")
    @Override
    public void checkContentMetadata(Blob blob) {
-      if (!org.jclouds.utils.TestUtils.isMacOSX()) {
-         super.checkContentMetadata(blob);
-      }
+      super.checkContentMetadata(blob);
    }
 
-   // Mac OS X HFS+ does not support UserDefinedFileAttributeView:
-   // https://bugs.openjdk.java.net/browse/JDK-8030048
+   @Test(dataProvider = "ignoreOnMacOSX")
    @Override
    protected void checkContentDisposition(Blob blob, String contentDisposition) {
-      if (!org.jclouds.utils.TestUtils.isMacOSX()) {
-         super.checkContentDisposition(blob, contentDisposition);
-      }
+      super.checkContentDisposition(blob, contentDisposition);
    }
 
-   // Mac OS X HFS+ does not support UserDefinedFileAttributeView:
-   // https://bugs.openjdk.java.net/browse/JDK-8030048
+   @Test(dataProvider = "ignoreOnMacOSX")
    @Override
    protected void validateMetadata(BlobMetadata metadata) throws IOException {
-      if (!org.jclouds.utils.TestUtils.isMacOSX()) {
-         super.validateMetadata(metadata);
-      }
+      super.validateMetadata(metadata);
    }
 
-   // Mac OS X HFS+ does not support UserDefinedFileAttributeView:
-   // https://bugs.openjdk.java.net/browse/JDK-8030048
    @Test(dataProvider = "ignoreOnMacOSX")
    @Override
    public void testCreateBlobWithExpiry() throws InterruptedException {
       super.testCreateBlobWithExpiry();
    }
+
+   @Test(dataProvider = "ignoreOnMacOSX")
+   @Override
+   public void testPutByteSource() throws Exception {
+      super.testPutByteSource();
+   }
+
+   @Test(dataProvider = "ignoreOnMacOSX")
+   @Override
+   public void testPutInputStream() throws Exception {
+      super.testPutInputStream();
```

Comment:
This doesn't seem like the right approach.  Instead this should override `checkContentMetadata` and similar.